src/frontend/contexts/auth-context.js

```diff
@@ -0,0 +1,48 @@
+import { createContext, useContext, useReducer } from "react";
+import { authActions } from "../reducer";
+
+const authContext = createContext(null);
+
+const useAuthContext = () => useContext(authContext);
+
+function AuthProvider({ children }) {
+	const localstoragetoken = localStorage.getItem("token");
+
+	const authReducer = (state, action) => {
+		switch (action.type) {
+			case authActions.FIRST_NAME:
```

Comment:
first name, last name, email and password can be stored as an object.

Reply:
Yes added